exercises/practice/queen-attack/queen_attack.go

```diff
@@ -1 +1,5 @@
 package queenattack
+
+func CanQueenAttack(whiteQueen, blackQueen string) (bool, error) {
```

Comment:
Wasn't sure what these parameters were supposed to represent. The example solution has the following:
https://github.com/exercism/go/blob/67cb0d27a99524dec7b8467f2b825a061e7a652c/exercises/practice/queen-attack/.meta/example.go#L5

Is there a more accurate name we could use?

Reply:
Maybe `whitePosition` and `blackPosition`, since the strings are the positions of the queens?

Reply:
I like those names better 👍 
Writing this as a suggestion so I can add it to the batch
```suggestion
func CanQueenAttack(whitePosition, blackPosition string) (bool, error) {
```